floragunncom/search-guard-kibana-plugin
# Allow components to handle a service .get() method 404 error

This closes https://floragunn.atlassian.net/browse/ITT-2213

To test this PR do the following
1) Create or clone a role on **Roles** page
2) Create a corresponding mapping on **Role Mappings** page
3) Delete the created role on **Roles** page
4) Go back to **Role Mappings** page
5) See there is no error toasted
6) See **No corresponding role** error in the corresponding mapping Name cell

![Screenshot 2019-06-24 at 12 00 42](https://user-images.githubusercontent.com/5389745/60010715-a6314700-9678-11e9-9742-6406c3138240.png)
